package.json

```diff
@@ -248,7 +248,6 @@
 		"design-system:dev": "echo \"Please use storybook:dev instead.\"",
 		"playground:build": "npm run storybook:build",
 		"playground:dev": "echo \"Please use storybook:dev instead.\"",
-		"preenv": "npm run check-engines",
 		"env": "wp-scripts env",
 		"wp-env": "packages/env/bin/wp-env"
```

Comment:
Have you considered making “npm run env” the primary way to reference it everywhere?

Reply:
That would be confusing because the binary is `wp-env`. It's better to be aligned with what people use when invoking a global installation.

Reply:
I’m not so convinced that the name of binary file should be the main reason to decide. If they know about the underlying tool, they can always call `npx wp-env` that is shorter. 
`env` is simply shorter and I’m sure that  `wp-env` is something that is largely unfamiliar for folks that contribute.

Anyway, whatever you pick is fine here 🤷‍♂️

Reply:
People have already been using `npm run wp-env`, and it's in all the docs.